ansible/templates/var/www/koala/.rbenv-vars.j2

```diff
@@ -7,6 +7,7 @@ DATABASE_SOCKET=/var/run/mysqld/mysqld.sock
 
 SECRET_KEY_BASE={{ secret_koala.rails_secret }}
 DEVISE_SECRET={{ secret_koala.devise_secret }}
+OIDC_SIGNING_KEY={{ secret_koala.oidc_signing_key_location }}
```

Comment:
If this is for setting the _location_ of the key, the name should be something like `OIDC_SIGNING_KEY_PATH`.